python3.6/PyZAPI/development.ini

```diff
@@ -0,0 +1,70 @@
+###
```

Comment:
Agreed, we do the same here with our config files.
Will make the changes, and add the ini files to gitignore